content/sensu-go/5.14/installation/install-sensu.md

```diff
@@ -212,15 +212,15 @@ sensuctl configure -n \
 --url 'http://127.0.0.1:8080'
 {{< /highlight >}}
 
-Here the `-n` flag triggers non-interactive mode.
-Run `sensuctl config view` to see your user profile.
+Here the `-n` flag triggers non-interactive mode. Run `sensuctl config view` to see your user profile.
```

Comment:
Were these new line changes intention? If so, is this something we want to put to the rest of the install sensu pages later?

Reply:
Oh -- the single hard return causes words/sentences to run together in the PDF conversion -- the return is removed and it would look like "...mode.Run..." in the PDF. I noticed it here so I removed it (but I can put it back if it's a problem).

I'm not sure if it will end up being a doc-wide backward-looking chore or just something we fix going forward.

Reply:
Ah, the joys. Yeah, we can decide that later. Thanks!